db/models/order.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const {STRING, INTEGER, DOUBLE} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  orderNumber: STRING,
```

Comment:
orderNumber needs to be repeated in the table.  Orders table lists items in orders.  1 record = 1 item in order.  Multiple records could = 1 order.